person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] The `rentals` here should be set to getter and setter - attr_accessor.

Reply:
With all due respect, @julie-ify, I disagree, `:rentals` is a mutable object that has its own methods to handle data internally, if we move it to `attr_accessor`, it can be overridden which should never happen.